apps/meet-app/webapp/src/view/meetings/panel/createMeeting.tsx

```diff
@@ -46,7 +46,11 @@ import { DatePicker, TimePicker, LocalizationProvider } from "@mui/x-date-picker
 // Extend dayjs with timezone and UTC functionality
 dayjs.extend(utc);
 dayjs.extend(timezone);
-
+const defaultTz = Intl.DateTimeFormat().resolvedOptions().timeZone;
+const safeTz    = (tz?: string) => tz && tz.trim() ? tz : defaultTz;
+const nowIn = (tz?: string) => dayjs().tz(safeTz(tz));
+const asTzDate = (d: Dayjs, tz?: string) =>
+  dayjs.tz(d.format("YYYY-MM-DD"), safeTz(tz));
```

Comment:
Shall we update the logic as follows to improve accuracy and cover all edge cases?

1. Retrieve the selected time zone from the input.
2. Get the **current date and time** based on the selected time zone and store it as `currentDateTime`.
3. Pass this `currentDateTime` to the date picker as the `minDate` to validate the selected date.
4. For **start time** validation:
   - Combine the selected date and picked start time into a single DateTime object.
   - Use Day.js’s `isBefore()` function to check whether the selected DateTime is before the `currentTime` in the selected time zone.

This approach will ensure accurate validation based on the user’s selected time zone and handle all possible edge cases.